engine/schema/src/main/resources/META-INF/db/schema-41200to41300.sql

```diff
@@ -170,6 +170,130 @@ INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid, hypervisor_type, hypervi
 ALTER TABLE `cloud`.`service_offering_details` CHANGE COLUMN `value` `value` TEXT NOT NULL;
 
 ALTER TABLE `cloud`.`vpc_offerings` ADD COLUMN `sort_key` int(32) NOT NULL default 0 COMMENT 'sort key used for customising sort method';
```

Comment:
You are still adding these to schema-41200to41300.sql instead of schema-41300to41400.sql !

Reply:
I think you mean this: 
-- PR#3186 Add possibility to set MTU size for NIC
ALTER TABLE `cloud`.`nics` ADD COLUMN `mtu` smallint (6) COMMENT 'MTU size for the interface';

DROP VIEW IF EXISTS `cloud`.`user_vm_view`;
CREATE VIEW `cloud`.`user_vm_view` AS 
...
and I moved that to the current schema.